internal/rukpak/source/containers_image.go

```diff
@@ -171,6 +194,35 @@ func (i *ContainersImageRegistry) Cleanup(_ context.Context, bundle *BundleSourc
 	return deleteRecursive(i.bundlePath(bundle.Name))
 }
 
+func (i *ContainersImageRegistry) checkUpdatedConfiguration() error {
+	i.checkConfigLock.Lock()
+	defer i.checkConfigLock.Unlock()
+	dirEntries, err := os.ReadDir(containersConfigFolder)
+	if err != nil && !os.IsNotExist(err) {
+		// Ignore not found errors
+		return fmt.Errorf("could not read registries configuration directory, error: %w", err)
+	}
+	for _, entry := range dirEntries {
+		if entry.Type().IsRegular() || !strings.HasPrefix(entry.Name(), "..") {
+			// Skip 'normal' entries
+			continue
+		}
+		// Trim the leading '..' for symlinks
+		trimmed := strings.TrimLeft(entry.Name(), "..")
+		t, err := time.Parse(mountedSymlinkTimestampFormat, trimmed)
+		if err != nil {
+			continue
+		} else if t != i.RegistriesConfTimestamp {
```

Comment:
it's generally safer to use `t.Equal()` rather than `==`, see: https://pkg.go.dev/time#hdr-Monotonic_Clocks